source/source_test.go

```diff
@@ -74,10 +79,15 @@ func TestSource_Configure_allFieldsSuccess(t *testing.T) {
 	is.NoErr(err)
 	is.Equal(s.config, config.Config{
 		Configuration: common.Configuration{
-			DSN:        testDSN,
-			Table:      testTable,
-			KeyColumns: []string{"id", "name"},
+			DSN: testDSN,
 		},
+		Tables: func() map[string]config.TableConfig {
+			tables := make(map[string]config.TableConfig)
+			tables[testTable] = config.TableConfig{OrderingColumn: "created_at", KeyColumns: []string{"id", "name"}}
+
+			return tables
```

Comment:
Nitpick: I think this can be shortened to:

```go
return map[string]config.TableConfig{
    testTable: config.TableConfig{OrderingColumn: "created_at", KeyColumns: []string{"id", "name"}}
}
```